src/models/IssueTableStorage.gs

```diff
@@ -0,0 +1,127 @@
+// Node required code block
+const Storage_ = require('../Storage.gs').Storage_;
+// End of Node required code block
+
+/**
+ * @file Contains class used to persist IssueTable data and access it.
+ */
+
+/**
+ * Creates a new IssueTableIndex_ instance, which is used to persist IssueTables and related information.
+ * 
+ * @constructor
+ */
+function IssueTableIndex_() {
+  this.storage_ = new Storage_('jst_tables', PropertiesService.getScriptProperties() || {});
+  this.index_ = {};  // only index of all table ids
+  this.tables_ = {}; // actual array/object of IssueTables
+}
+
+/**
+ * Adding a IssueTable to storage.
+ * @param {IssueTable_} IssueTable    The IssueTable object to store in script storage
+ * @return {IssueTableIndex_}    Allow chaining
+ */
+IssueTableIndex_.prototype.addTable = function(IssueTable) {
```

Comment:
Why is the underscore used for the class names?
I don't think this is necessary for classes - only functions?

Reply:
A class is nothing else then a function. The underscore is needed for everything which shall not be exposed to the user for Macros or other access. See google docs. IE: Googles "Storage_" class is declared as private this way too.

I would actually suggest to do *everything* private this way, unless it is really expected to be accessible from outside by users or other scripts.

Reply:
We should decide - I'm totally happy with a decision - but don't want to chase a moving target on the code standards. The code base is pretty mixed up with a mix of functional / classes / folder structure so good to make a decision.

From my perspective the "_" is to stop the function being called from the sheet and unintentional side effects

Reply:
So if i interpret google docs correctly, i would conclude to use the trailing "_" is most cases.
If you agree; this is the decision then.

Reply:
@ljay79 - I will do this now.